src/agni/server/manager/LoginManager.java

```diff
@@ -18,7 +18,7 @@ public LoginManager(InfoSender infoSender,
     @Override
     public void loginRequest(String ip, String user, String password) {
         // TODO Auto-generated method stub
-        
+    	
```

Comment:
Probably remove this change in this PR to make sure we don't get merge conflicts later.